cmd/family-migrator/main.go

```diff
@@ -363,6 +383,35 @@ func askExecutionSteps(kongCtx *kong.Context, plan *migration.Plan, opts *Option
 	}
 }
 
+func getFamilyProviderVersions(kongCtx *kong.Context, providerName string) []interface{} {
+	var versions []interface{}
+	resp, err := http.Get(fmt.Sprintf("https://api.upbound.io/v1/packageMetadata/upbound/%s", monolithicToFamily[providerName]))
+	if err != nil {
+		fmt.Println("No suitable version found in the marketplace for the specified provider. Please check manually.")
```

Comment:
Do we just print a message here, shouldn't this also be treated as a validation error? I think because the user can select from a predefined set, we don't expect to end up here, right?

Reply:
When we could not get the versions we wanted from the marketplace for some reason, it seemed more correct to me that the migration process was not affected. Because the user still enters the input herself. We're trying to do something like making the user's job easier and guide her while listing these possible versions. Do you have a strong opinion on this?

Reply:
Agreed. Maybe we had better incorporate a logging framework with debug logging support to improve the UX. Not a blocker for this PR, a comment for you to consider in a future PR.